corehq/toggles.py

```diff
@@ -1744,3 +1744,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_CUSTOM,
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
+
+IMPROVED_ASR_REPORT = StaticToggle(
```

Comment:
Any reason to not add this to the other toggle you created in  a previous PR?

Reply:
This will allow independent release, QA for both.